easybuild/easyconfigs/f/FIAT/FIAT-2019.1.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonBundle'
+
+name = 'FIAT'
+version = '2019.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://bitbucket.org/fenics-project/fiat'
+description = """The FInite element Automatic Tabulator (FIAT) supports
+generation of arbitrary order instances of the Lagrange elements on
+lines, triangles, and tetrahedra. It is also capable of generating
+arbitrary order instances of Jacobi-type quadrature rules on the same
+element shapes."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('sympy', '1.6', {
```

Comment:
@akesandgren We have an easyconfig for `sympy` 1.5.1, might make more sense to use that as a proper dep?

Unless you really need 1.6, then we can add a new easyconfig for `sympy` 1.6 (since 1.5.1 is not used as a dep yet anywhere, so that's still feasible).